mappings/net/minecraft/resource/featuretoggle/FeatureSet.mapping

```diff
@@ -0,0 +1,29 @@
+CLASS net/minecraft/class_7699 net/minecraft/resource/featuretoggle/FeatureSet
+	FIELD field_40172 MAX_FEATURE_FLAGS I
+	FIELD field_40173 EMPTY Lnet/minecraft/class_7699;
+	FIELD field_40174 universe Lnet/minecraft/class_7700;
+	FIELD field_40175 featuresMask J
+	METHOD <init> (Lnet/minecraft/class_7700;J)V
+		ARG 1 universe
+		ARG 2 featuresMask
+	METHOD equals (Ljava/lang/Object;)Z
+		ARG 1 featureSet
```

Comment:
```suggestion
		ARG 1 o
```
`o` is the convention in yarn.

> "o" for the parameter of `equals(Ljava/lang/Object;)Z` methods

https://github.com/FabricMC/yarn/blob/349270214cfb36eb3cb84d7846070a431283e9cd/CONVENTIONS.md#abbreviations